Sources/MapboxNavigation/Constants.swift

```diff
@@ -25,6 +30,11 @@ public var RouteLineWidthByZoomLevel: [Double: Double] = [
  */
 public var NavigationMapViewMinimumDistanceForOverheadZooming: CLLocationDistance = 200
 
+/**
+ Attribute name for the route line that is used for identifying restricted areas along the route.
+ */
+public let RestrictedRoadClassAttribute = "isRestrictedRoad"
```

Comment:
There are also few convenience constants exposed to user, like table of route line widths depending on zoom level (`RouteLineWidthByZoomLevel`). The only value for restricted areas display I could think of is `dash array attributes` which may be potentially useful, but I doubt it since if user is implementing his own layers he won't need to align dashing parameters to anything mapbox-provided.